app/views/searches/search.html.erb

```diff
@@ -0,0 +1,72 @@
+<div class="row">
+  <% if !@provider.nil? %>
+  <div class="jumbotron">
+    <h1>
+      <%= @provider.capitalize %> Search Results
+    </h1>
+  </div>
+  <% else%>
+    <div class="jumbotron">
+      <h1>Please select Twitter or Vimeo</h1>
+    </div>
+<!-- </div> -->
+<% end %>
+<% if !@results.nil? %>
+  <% @results.in_groups_of(3, false) do |group| %>
+    <div class="row">
+      <% group.each do |result| %>
+        <div class="col-sm-4 col-md-4 creator-col">
+          <% if @provider == "twitter" %>
+            <% picture = result.profile_image_uri if !result.profile_image_uri.nil? %>
+          <% elsif @provider == "vimeo" %>
```

Comment:
This logic probably belongs in your controller. Set `@picture` and then access it here. Views are for display -- single responsibility!

Also, you can combine your conditionals here to avoid the nested if on the next line, e.g.,
``` 
elsif @provider == "vimeo"  && result["pictures"] 
  picture = result["pictures"]["sizes"][2]["link"]  # Why so nested, btw? Is this the structure from the API?
elsif @provider == "vimeo"  && result["pictures"] .nil?
  picture = "vimeo_default_two.png"
end
```

Same applies to the if block that starts on line 30.